ios/FluentUI/Command Bar/CommandBarButton.swift

```diff
@@ -55,12 +55,12 @@ class CommandBarButton: UIButton {
         isEnabled = item.isEnabled
         isSelected = isPersistSelection && item.isSelected
 
-        if item.iconImage == nil {
-            setTitle(item.title, for: .normal)
-            if let font = item.titleFont {
-                titleLabel?.font = font
-            }
-        }
+        // always update icon and title as we only display one; we may alterenate between them, and the icon may also change
+        let iconImage: UIImage? = item.iconImage
+        setImage(iconImage, for: .normal)
+        setTitle(((iconImage != nil) ? nil : item.title), for: .normal)
```

Comment:
nit: parenthesis style

`setTitle(iconImage != nil ? nil : item.title, for: .normal)`

Reply:
Updated (the second brackets were added by Xcode, the first were intended for readability)